Revision: 5757afe957351b46356dc2c8ca0ccfe9bbae5f7d
Patch-set: 1
File: runtime/entrypoints/entrypoint_utils.h

661:4-661:73
Mon Jun 02 16:52:25 2014 +0000
Author: Andreas Gampe <1041833@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 3fef0c01_e47ec2f1
Bytes: 10
Duplicate?

661:4-661:73
Mon Jun 02 18:54:53 2014 +0000
Author: Sebastien Hertz <1029223@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 3fef0c01_e47ec2f1
UUID: ffd8146d_62251e12
Bytes: 4
Done

File: runtime/instrumentation.h

239:22-239:25
Mon Jun 02 16:52:25 2014 +0000
Author: Andreas Gampe <1041833@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: dfd51034_0171bab9
Bytes: 5
Typo.

239:22-239:25
Mon Jun 02 18:54:53 2014 +0000
Author: Sebastien Hertz <1029223@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: dfd51034_0171bab9
UUID: ffd8146d_221f16be
Bytes: 4
Done

File: runtime/quick_exception_handler.cc

154:4-154:8
Mon Jun 02 18:54:53 2014 +0000
Author: Sebastien Hertz <1029223@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: ffd8146d_c20df29b
Bytes: 124
Finally I don't think it's true. We must clear the flag if we're catching the exception, whatever the state of this flag is.

